main.cpp

```diff
@@ -83,68 +116,112 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
+float rentalPrice(unsigned int rentalDuration, unsigned int carType = 0, unsigned int insurance = 0)
+
+{
+    ignoreUnused(rentalDuration, carType, insurance);  
 
+    return {};
+}
 /*
  2)
  */
+int carReference(int carColor = 0, int carLicenseNumber = 00000, unsigned int carConstructionYear = 2015)
+{
+    ignoreUnused(carColor, carLicenseNumber, carConstructionYear);  
+    return {};
+}
 
 /*
  3)
  */
-
+int jarContentAirExposure(bool jarOpen = false,  unsigned int sizeOfjarOpening = 0,int durationOfjarOpening = 0)
+{
+    ignoreUnused(jarOpen, sizeOfjarOpening, durationOfjarOpening);  
+    return {};
+}
 /*
  4)
  */
-
+int requiredAmountPasta(unsigned int numberOfPeople = 0)
+{
+    ignoreUnused(numberOfPeople);  
+    return {};
+}
 /*
  5)
  */
-
+bool eatSandwich(int speed, int biteSize = 4)
+{
+    ignoreUnused(speed, biteSize);  
+    return {};
+}
 /*
  6)
  */
-
+bool trigger(unsigned int midiChannel = 0, unsigned int velocity = 127)
+{
+    ignoreUnused(midiChannel, velocity);  
+    return {};
+}
 /*
  7)
  */
-
+float lfo1(float depth = 0, float rate = 5, char waveform = 0)
+{
+    ignoreUnused(depth, rate, waveform);  
+    return {};
+}
 /*
  8)
  */
-
+bool fermenting(bool brettanomyces = false, bool pitchedYeast = true, bool barrel = false, bool bottleConditioned = true)
+{
+    ignoreUnused(brettanomyces, pitchedYeast, barrel, bottleConditioned);  
+    return {};
+}
 /*
  9)
  */
-
+int time(int position = 0, bool summerTime = true)
+{
+    ignoreUnused(position, summerTime);  
+    return {};
+}
 /*
  10)
  */
+int hangry(int hunger, int anger)
+{
+    ignoreUnused(hunger, anger);  
+    return {};
+}
 
 int main()
 {
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    rentalPrice(15);
     //2)
-    
+    carReference(5,16543,2018);
     //3)
-    
+    jarContentAirExposure();
     //4)
-    
+    requiredAmountPasta(2);
     //5)
-    
+    eatSandwich(10,6);
     //6)
-    
+    trigger(0,89);
     //7)
-    
+    lfo1(55, 0.125f, 7);
     //8)
-    
+    fermenting(1,0,1,1);
     //9)
-    
+    time(1, true);
     //10)
-    
+    hangry(5,5);
```

Comment:
some of these return values.   you need to follow the example on line 203 and store what they return, and then pass it to `ignoreUnused()` at the end.
- [ ] revise